test/form/external-imports-custom-names/_expected/cjs.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-var $ = require('jquery');
-$ = 'default' in $ ? $['default'] : $;
+function _interopRequire (id) { var ex = require(id); return 'default' in ex ? ex['default'] : ex; }
+
+var $ = _interopRequire('jquery');
```

Comment:
I think this should be done in two steps like it was before, otherwise using `browserify` or the like on the output won't really work. Unless I'm missing something?